### DISCUSSION TITLE
Allowing GitHub Copilot on Kaspersky Firewall

### DISCUSSION THREAD

**Discussion by RianKoja** [community/community#30411] [Copilot Conversations]
My company policies require that developers work on virtual machines with no internet connection. It would be absolutely forbidden for these machines to have access to Github.com. However, I've been told that some IP addresses can and are whitelisted, on an IP address based (URLs are no good). If GitHub Copilot used fixed IP addresses (like Bloomberg does), that would allow using copilot on these machines.

Is that the case? Does anyone know if GitHub Copilot uses fixed IP addresses? Or how to contact support for this product?

Best regards

**Comment by iamcjwilliams** [comment]
Hi there, We do not use fixed IP addresses. We use dynamic IP addresses with a domain that resolves to them
